packages/cli/src/commands/setup/shadowenv/shadowenv.js

```diff
@@ -0,0 +1,77 @@
+import fs from 'fs'
+import path from 'path'
+
+import boxen from 'boxen'
+import Listr from 'listr'
+
+import { getPaths, writeFile } from '../../../lib'
+import c from '../../../lib/colors'
+
```

Comment:
```suggestion

export const command = 'cli-alias'
export const aliases = ['shadowenv']
export const description =
  "Set up CLI command aliasing, e.g. 'yarn rw' --> 'rw', using Shadowenv. Note: this can also be used to create project-local env var shadowing. For more info: https://shopify.github.io/shadowenv/"

```

cc @jtoar 

Ok, I'm throwing several ideas into the mix here. First off, we do need a description for discovery and when user's pass `rw setup --help`. Secondly, I'm stuck on the command name — no one will know what shadowenv does until they read the description. Also, it does more than just  alias (a whole lot more). So in this snippet I suggest:
- calling the command 'cli-alias' (which means we should probably rename the file
- giving the command an alias (which will output in --help) basically for SEO and/or the case where people know of shadowenv

Open to discussion here. Want to keep it both clear (according to usage and intention) and simple.